NebulaPatcher/Patches/Dynamic/PlanetFactory_Patch.cs

```diff
@@ -13,21 +13,19 @@ namespace NebulaPatcher.Patches.Dynamic
     [HarmonyPatch(typeof(PlanetFactory))]
     class BuildFinally_patch
     {
-        [HarmonyPrefix]
-        [HarmonyPatch("AddPrebuildDataWithComponents")]
-        public static bool AddPrebuildDataWithComponents_Prefix(PlanetFactory __instance, PrebuildData prebuild)
+        [HarmonyPostfix]
+        [HarmonyPatch("AddPrebuildData")]
+        public static void AddPrebuildData_Prefix(PlanetFactory __instance, PrebuildData prebuild, ref int __result)
```

Comment:
Should rename the method to `AddPrebuildData_Postfix`

Reply:
fixed :)